### PR TITLE
Update install.yaml

### PR DESCRIPTION
**Pull request by alexef**
Match the latest available release.

### CODE REVIEW

File: manifests/install.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ metadata:
 spec:
   sources:
   - web:
-      url: https://github.com/argoproj-labs/rollout-extension/releases/download/v0.1.0/extension.tar
+      url: https://github.com/alexef/rollout-extension/releases/download/v0.2.2/extension.tar
```
Comment (rbreeze) [inline manifests/install.yaml:10]: I think we'd like to keep this in the `argoproj-labs` repository

File: README.md [hunk 2]
```diff
@@ -12,6 +12,6 @@ The project introduces the Argo Rollout dashboard into the Argo CD Web UI.
 
 ```
 kubectl apply -n argocd \
-    -f https://raw.githubusercontent.com/argoproj-labs/rollout-extension/v0.1.0/manifests/install.yaml
+    -f https://raw.githubusercontent.com/argoproj-labs/rollout-extension/master/manifests/install.yaml
```
Comment (rbreeze) [inline README.md:15]: We should keep this as a fixed version (instead of `master`) and instead update the README on each release with the new version

**Review by rbreeze — CHANGES_REQUESTED** [review]

File: manifests/install.yaml (hunk 1 above)
Comment (alexef) [reply manifests/install.yaml:10]: sorry, I must have pushed on the wrong branch. let me revert this commit

Comment (alexef) [reply manifests/install.yaml:10]:
```suggestion
      url: https://github.com/argoproj-labs/rollout-extension/releases/download/v0.2.1/extension.tar
```

**Review by rbreeze — APPROVED** [review]

**Pull request merged**